template/web/.env.local.example

```diff
@@ -1,6 +0,0 @@
-NEXT_PUBLIC_GOOGLE_ANALYTICS_ID=
```

Comment:
why did you deleted this?

Reply:
I renamed this file locally from `.env.local.example` to` `.env.local`

Reply:
Added example env file back in